bento_wes/db.py

```diff
@@ -155,41 +155,95 @@ def finish_run(
                 ),
             )
 
-    def update_stuck_runs(self):
-        # Update all runs that have "stuck" states to have an error state instead on restart. This way, systems don't
-        # get stuck checking their status, and if they're in a weird state at boot they should receive an error status
-        # anyway.
-
-        event_bus = get_flask_event_bus()
-
-        c = self.cursor()
-        logger: logging.Logger = current_app.logger
-
-        c.execute("SELECT id FROM runs WHERE state IN (?, ?)", (states.STATE_INITIALIZING, states.STATE_RUNNING))
-        stuck_run_ids: list[sqlite3.Row] = c.fetchall()
+    def update_stuck_runs(self) -> None:
+        """
+        On process boot, convert initializing/running states into system error so
+        the UI/backend doesn't wait on orphaned work.
+        """
+        self.c.execute(
+            "SELECT id FROM runs WHERE state IN (?, ?)",
+            (states.STATE_INITIALIZING, states.STATE_RUNNING),
+        )
+        stuck_run_ids: list[sqlite3.Row] = self.c.fetchall()
 
         for r in stuck_run_ids:
-            run = self.get_run_with_details(c, r["id"], stream_content=True)
+            run = self.get_run_with_details(self.c, r["id"], stream_content=True)
```

Comment:
given our approach to cursors now where the objects never get reused, there's not much point in this being a passed argument.